src/Components/Patient/ManagePatients.tsx

```diff
@@ -478,7 +471,7 @@ export const PatientManager = () => {
             className={`absolute inset-y-0 left-0 flex h-full w-1 items-center rounded-l-lg transition-all duration-200 ease-in-out group-hover:w-5 ${categoryClass}`}
           >
             <span className="absolute -inset-x-32 inset-y-0 flex -rotate-90 items-center justify-center text-center text-xs font-bold uppercase tracking-widest opacity-0 transition-all duration-200 ease-in-out group-hover:opacity-100">
-              {category ? PatientCategoryDisplayText[category] : "UNKNOWN"}
+              {category ? category.toUpperCase() : "UNKNOWN"}
```

Comment:
`uppercase` tailwind class is already applied to the element.

```suggestion
              {category || "UNKNOWN"}
```